tests/test_utils.py

```diff
@@ -343,13 +386,14 @@
                 pd.Series([1.], audformat.filewise_index('f1')),
                 pd.Series([2.], audformat.filewise_index('f1')),
             ],
+            False,
             None,
             marks=pytest.mark.xfail(raises=ValueError),
         ),
     ],
 )
-def test_concat(objs, expected):
-    obj = utils.concat(objs)
+def test_concat(objs, overwrite, expected):
```

Comment:
I think you should also add a few tests that combine three objects.

Reply:
we now have:

```python
# combine series and data frame
        (
            [
                pd.Series(
                    [1., 2.],
                    audformat.filewise_index(['f1', 'f2']),
                    name='c1',
                ),
                pd.Series(
                    ['a', np.nan, 'd'],
                    audformat.filewise_index(['f1', 'f2', 'f4']),
                    name='c2',
                ),
                pd.DataFrame(
                    {
                        'c1': [np.nan, 3.],
                        'c2': ['b', 'c'],
                    },
                    audformat.segmented_index(['f2', 'f3']),
                ),
            ],
            False,
            pd.DataFrame(
                {
                    'c1': [1., 2., 3., np.nan],
                    'c2': ['a', 'b', 'c', 'd']
                },
                audformat.segmented_index(['f1', 'f2', 'f3', 'f4']),
            ),
        ),
```